### PR TITLE
(Changes made) Add Deep Reinforcement Learning Course from Hugging Face

### PR DESCRIPTION
**Pull request by Anant-mishra1729**
## What does this PR do?
[Deep Reinforcement Learning Course](https://huggingface.co/learn/deep-rl-course/unit0/introduction)

### Description
Free Deep Reinforcement Learning Course from Hugging Face community

### Why is this valuable (or not)?
Beginner-friendly course, providing a solid foundation from the basics to the state-of-the-art methods in deep RL.

### How do we know it's really free?
The course is listed as free on the community website

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by Thenlie — APPROVED** [review]

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -523,6 +523,7 @@
 * [Deep Learning With Tensorflow 2.0 and Keras](https://youtube.com/playlist?list=PLeo1K3hjS3uu7CxAacxVndI4bE_o3BDtO) - codebasics
 * [Deep Multi-Task and Meta Learning](https://cs330.stanford.edu) - Chelsea Finn (Stanford University)
 * [Deep Reinforcement Learning](http://rail.eecs.berkeley.edu/deeprlcourse/) - Sergey Levine
+* [Deep Reinforcement Learning Course - Hugging Face](https://huggingface.co/learn/deep-rl-course/unit0/introduction)
```
Comment (eshellman) [inline courses/free-courses-en.md:526]:
```suggestion
* [Deep Reinforcement Learning Course](https://huggingface.co/learn/deep-rl-course/unit0/introduction) (Hugging Face)
```

Comment (eshellman) [reply courses/free-courses-en.md:526]: when you've made changes, please add `(changes made)` to the title of your PR

Comment (Anant-mishra1729) [reply courses/free-courses-en.md:526]: > when you've made changes, please add `(changes made)` to the title of your PR
Changed the PR title.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**